docs/Measuring_Resonances.md

```diff
@@ -65,29 +65,29 @@ the system that may damage the electronics.
 
 ## Software installation
 
-Note that resonance measurements and shaper auto-calibration require additional
-software dependencies not installed by default. First, you will have to run on
-your Raspberry Pi the following command:
+Because setting up the software for auto-calibration takes significant
+time and resources, it is not installed by default. Run the following
+script to setup `numpy` and `matplotlib` to get started:
 ```
-~/klippy-env/bin/pip install -v numpy
+~/klipper/scripts/setup-shaper-calibrate.sh
 ```
-to install `numpy` package. Note that, depending on the performance of the
+Note that, depending on the performance of the
 CPU, it may take *a lot* of time, up to 10-20 minutes. Be patient and wait
 for the completion of the installation. On some occasions, if the board has
-too little RAM, the installation may fail and you will need to enable swap.
+too little RAM, the installation may fail, and you will need to
+[enable swap](https://nebl.io/neblio-university/enabling-increasing-raspberry-pi-swap/).
 
-Next, run the following commands to install the additional dependencies:
-```
-sudo apt update
-sudo apt install python-numpy python-matplotlib
-```
+## Hardware Configuration
+You can connect your ADXL345 to any spi bus supported by Klipper. Regular printer
+control boards, dedicated boards, and the Raspberry Pi mcu are all options that have
+their own advantages and disadvantages.
 
-Afterwards, check and follow the instructions in the
-[RPi Microcontroller document](RPi_microcontroller.md) to setup the
-"linux mcu" on the Raspberry Pi.
-
-Make sure the Linux SPI driver is enabled by running `sudo
-raspi-config` and enabling SPI under the "Interfacing options" menu.
+### Raspberry Pi MCU
+The reccomended way to connect the sensor to the printer in a permenant installation
```

Comment:
A couple of typos here: 'reccomended' and 'permenant'. BTW, I wouldn't require a permanent installation. I'd just write something along the lines of "The recommended way to connect the sensor to the printer is to connect it directly to the SPI bus on the Raspberry Pi, as per [wiring](#wiring) instructions. Then, check and follow ..."

BTW, "## Wiring" part only describes wiring to the RPi. If you want to keep the sections about connecting the ADXL345 to the printer MCUs, I'd update that section too somehow. Otherwise, the user may be confused as to how the ADXL345 must be physically connected to the MCU.

Reply:
what I mentioned in the comment on line 81 should address this